### PR TITLE
CASMINST-4551: Linting

### PR DESCRIPTION
**Pull request by mharding-hpe**
## Summary and Scope

Classic MitchLint (TM).

## Issues and Related PRs

* csm-1.2 PR: https://github.com/Cray-HPE/docs-csm/pull/1516
* csm-1.0 PR: https://github.com/Cray-HPE/docs-csm/pull/1519

## Testing

Testing your patience since 1977.

## Risks and Mitigations

Nope

## Pull Request Checklist

- [X] License file intact
- [X] Target branch correct

### CODE REVIEW

File: operations/power_management/Power_On_and_Boot_Compute_Nodes_and_User_Access_Nodes.md [hunk 1]
```diff
@@ -8,24 +8,25 @@ This procedure boots all compute nodes and user access nodes \(UANs\) in the con
 
 * All compute cabinet PDUs, servers, and switches must be powered on.
 * The Slingshot Fabric is up and configured.
-    Refer to the following documentation for more information on how to bring up the Slingshot Fabric:
-    -  The *Slingshot Administration Guide* PDF for HPE Cray EX systems.
-    -  The *Slingshot Troubleshooting Guide* PDF.
-* An authentication token is required to access the API gateway and to use the `sat` command. See the "SAT Authentication" section of the HPE Cray EX System Admin Toolkit (SAT) product stream documentation (S-8031) for instructions on how to acquire a SAT authentication token.
+  Refer to the following documentation for more information on how to bring up the Slingshot Fabric:
+  * The *Slingshot Administration Guide* PDF for HPE Cray EX systems.
```
Comment (nrockershousen) [inline operations/power_management/Power_On_and_Boot_Compute_Nodes_and_User_Access_Nodes.md:12]:
```suggestion
  * The *HPE Slingshot Operations Guide* PDF for HPE Cray EX systems.
```
This is the name on the HPE support center.

File: operations/power_management/Power_On_and_Boot_Compute_Nodes_and_User_Access_Nodes.md [hunk 2]
```diff
@@ -8,24 +8,25 @@ This procedure boots all compute nodes and user access nodes \(UANs\) in the con
...
+  * The *Slingshot Troubleshooting Guide* PDF.
```
Comment (nrockershousen) [inline operations/power_management/Power_On_and_Boot_Compute_Nodes_and_User_Access_Nodes.md:13]:
```suggestion
  * The *HPE Slingshot Troubleshooting* PDF.
```
This is the name on the HPE support center.

File: operations/power_management/Power_On_Compute_and_IO_Cabinets.md [hunk 3]
```diff
@@ -40,40 +42,47 @@ After the CDU is switched on and healthy, the liquid-cooled PDU circuit breakers
...
+   there are no error LEDS or hardware failures.
+
+1. Bring up the Slingshot Fabric.
+   Refer to the following documentation for more information on how to bring up the Slingshot Fabric:
 
-1.  Bring up the Slingshot Fabric.
-    Refer to the following documentation for more information on how to bring up the Slingshot Fabric:
-    -  The *Slingshot Administration Guide* PDF for HPE Cray EX systems.
-    -  The *Slingshot Troubleshooting Guide* PDF.
+      * The *Slingshot Administration Guide* PDF for HPE Cray EX systems.
```
Comment (nrockershousen) [inline operations/power_management/Power_On_Compute_and_IO_Cabinets.md:84]:
```suggestion
      * The *HPE Slingshot Operations Guide* PDF for HPE Cray EX systems.
```
Missed these in my first review.

File: operations/power_management/Power_On_Compute_and_IO_Cabinets.md [hunk 4]
```diff
@@ -40,40 +42,47 @@ After the CDU is switched on and healthy, the liquid-cooled PDU circuit breakers
...
+      * The *Slingshot Troubleshooting Guide* PDF.
```
Comment (nrockershousen) [inline operations/power_management/Power_On_Compute_and_IO_Cabinets.md:85]:
```suggestion
      * The *HPE Slingshot Troubleshooting* PDF.
```

**Review by nrockershousen — APPROVED** [review]

**Review by rustydb — APPROVED** [review]

**Review by gbaker-hpe — APPROVED** [review]

File: operations/power_management/Power_On_Compute_and_IO_Cabinets.md (hunk 3 above)
Comment (mharding-hpe) [reply operations/power_management/Power_On_Compute_and_IO_Cabinets.md:84]: This got merged before I could incorporate it, so I've added it to another linting PR of mine:
https://github.com/Cray-HPE/docs-csm/pull/1520

File: operations/power_management/Power_On_Compute_and_IO_Cabinets.md (hunk 4 above)
Comment (mharding-hpe) [reply operations/power_management/Power_On_Compute_and_IO_Cabinets.md:85]: This got merged before I could incorporate it, so I've added it to another linting PR of mine:
https://github.com/Cray-HPE/docs-csm/pull/1520

**Pull request merged**